mvc/3.0/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Jakarta MVC 3.0 (Under Development)"
+date: 2023-05-17
+summary: "Release supporing Jakarta EE 11"
+---
+
+Jakarta MVC defines a standard for creating web applications following the action-based model-view-controller pattern.
+
+This release ensures alignment with Jakarta RESTful Web Services 4.0, removes the requirement to support the Facelets view engine, as well as minor features requested by the community.
+
+### New features, enhancements or additions
+<!-- List here -->
+* Switch CSRF default to implicit
+* Add accessor for form method overwrite field name
+* Replace org.eclipse.krazo.defaultViewFileExtension with API constant
+
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* Remove requirement for Facelets view engine
```

Comment:
Just for clarification, is this a backwards incompatible change or more akin to making it optional?

Reply:
This is backward incompatible. This is a case where we _could_ do a 2.x release with the deprecation, and then remove in 3.0. But in this case, we want to apply for relaxing that requirement from the upcoming policy to avoid "busy works". MVC is not _yet_ a part of the Platform, so the exception, in this case, sounds reasonable.

Reply:
Specs that are not part of a platform or profile should have more flexibility in making changes. I would say they free to make changes as the project sees fit. I'll bring this up in the ongoing discussion of what the Jakarta versioning requirements are at the spec committee.

Reply:
Maybe it would be good for existing users to mention why it's being removed and what the alternative is? Apologies if this is already mentioned somewhere and I missed it.